core/network/impl/protocols/block_announce_protocol.cpp

```diff
@@ -124,8 +124,8 @@ namespace kagome::network {
              peer_info.id);
 
     base_.host().newStream(
-        peer_info.id,
-        base_.protocol(),
+        peer_info,
+        base_.protocols(),
```

Comment:
why protocols? thought block announce has only one protocl id

Reply:
By design of newStream() - it no longer have version that takes single protocol.

Reply:
okay